src/NServiceBus.Core/obsoletes.cs

```diff
@@ -344,6 +344,34 @@ public static void RegisterEncryptionService(this EndpointConfiguration config,
             throw new NotImplementedException();
         }
     }
+
+    [ObsoleteEx(
+        Message = "Implement `IWantToRunWhenEndpointStartsAndStops` in the NServiceBus.Host package or NServiceBus.Host.AzureCloudService package instead",
+        RemoveInVersion = "7.0",
+        TreatAsErrorFromVersion = "6.0")]
+    public interface IWantToRunWhenBusStartsAndStops
+    {
+        Task Start(IMessageSession session);
```

Comment:
Either remove the methods or make them identical to the v5 interface https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/IWantToRunWhenBusStartsAndStops.cs

I vote for just removing them